packages/code-block/partials/code-tabs/partials/tabs-as-tabs/index.js

```diff
@@ -2,16 +2,20 @@ import React from 'react'
 import classNames from 'classnames'
 import s from './style.module.css'
 
-function TabsAsTabs({ tabLabels, activeTabIdx, setActiveTabIdx }) {
+function TabsAsTabs({ tabLabels, activeTabIdx, setActiveTabIdx, hasHeading }) {
   return (
-    <div className={s.tabsAsTabs}>
+    <div className={classNames(s.tabsAsTabs, { [s.hasHeading]: hasHeading })}>
```

Comment:
To achieve left-alignment when a `heading` is not present, but maintain the existing right-alignment when a `heading` is present, we add some styling for `TabsAsTabs` (and the space-limited alternate `TabsAsDropdowns`) which is applied conditionally based on the presence of a `heading`.